File: SDKs/hubapi.com/marketing/v3/java/lib/src/main/java/openapisdk/models/shared/BatchInputMarketingEventEmailSubscriber.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * BatchInputMarketingEventEmailSubscriber
 * List of marketing event details to create or update
**/
public class BatchInputMarketingEventEmailSubscriber {
    @JsonProperty("inputs")
    public MarketingEventEmailSubscriber[] inputs;
    public BatchInputMarketingEventEmailSubscriber withInputs(MarketingEventEmailSubscriber[] inputs) {
        this.inputs = inputs;
        return this;
    }
}

File: SDKs/hubapi.com/marketing/v3/python/src/sdk/models/shared/batchinputmarketingeventemailsubscriber.py
import dataclasses
from datetime import date, datetime
from marshmallow import fields
import dateutil.parser
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import marketingeventemailsubscriber as shared_marketingeventemailsubscriber


@dataclass_json
@dataclasses.dataclass
class BatchInputMarketingEventEmailSubscriber:
    r"""BatchInputMarketingEventEmailSubscriber
    List of marketing event details to create or update
    """
    
    inputs: list[shared_marketingeventemailsubscriber.MarketingEventEmailSubscriber] = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('inputs') }})
    
